tests/loading_datasets_test.py

```diff
@@ -19,7 +22,7 @@ def test_dataset_apply(gcp_setup, auth_tb_loading_datasets) -> None:
 
 
 @pytest.mark.dependency(depends=["test_dataset_apply"])
-def test_dataset_ready(gcp_setup, auth_tb_loading_datasets) -> None:
+def test_dataset_ready(auth_tb_loading_datasets) -> None:
```

Comment:
unnecessary to have this in the signature since `gcp_setup` declares `autouse=True` in its fixture decorator: `@pytest.fixture(scope="session", autouse=True)`